go/vt/vttablet/tabletmanager/vreplication/controller_plan_test.go

```diff
@@ -108,36 +141,40 @@ func TestControllerPlan(t *testing.T) {
 	}, {
 		in:  "update _vt.vreplication set state='Running', id = 2 where id = 1",
 		err: "id cannot be changed: id = 2",
-	}, {
-		in:  "update _vt.vreplication set state='Running'",
-		err: "invalid where clause:",
-	}, {
-		in:  "update _vt.vreplication set state='Running' where a = 1 and id = 2",
-		err: "invalid where clause: where a = 1 and id = 2",
-	}, {
-		in:  "update _vt.vreplication set state='Running' where a = 1",
-		err: "invalid where clause: where a = 1",
-	}, {
-		in:  "update _vt.vreplication set state='Running' where id > 1",
-		err: "invalid where clause: where id > 1",
-	}, {
-		in:  "update _vt.vreplication set state='Running' where id = 1.1",
```

Comment:
This I assume is still invalid. Same question as above.

Reply:
We accept this now because where clauses are passed through via a select statement.